crud/select/executor.lua

```diff
@@ -2,6 +2,8 @@ local errors = require('errors')
 
 local dev_checks = require('crud.common.dev_checks')
 local select_comparators = require('crud.compare.comparators')
+local compat = require('crud.common.compat')
+local has_keydef, keydef_lib = pcall(compat.require, 'tuple.keydef', 'key_def')
```

Comment:
I don't like the idea of using `pcall(compat.require)`. It can lead to ignoring some errors.
What about simple function `compare.exists('tuple.keydef', 'key_def')` that makes smth like this

```lua
    if module_cached ~= nil then
       return true
    end

    if package.search(module_name) then
        return true
    end

     if package.loaded[builtin_module_name] ~= nil then
         return true
    end

    return false
```